multiwallet.go

```diff
@@ -275,7 +280,7 @@ func (mw *MultiWallet) OpenWallets(startupPassphrase []byte) error {
 		return err
 	}
 
-	for _, wallet := range mw.wallets {
+	for _, wallet := range mw.walletsReadCopy() {
```

Comment:
Comment above applies here. Keep the mutex locked and prevent any other access to the map until the wallets are all opened.